server/tests/ballot_polling/test_minerva_ballot_polling.py

```diff
@@ -0,0 +1,335 @@
+import pytest
+from flask.testing import FlaskClient
+
+from ..helpers import *  # pylint: disable=wildcard-import
+
+
+@pytest.fixture
+def election_settings(client: FlaskClient, election_id: str):
+    settings = {
+        "electionName": "Test Election",
+        "online": False,
+        "randomSeed": "1234567890",
+        "riskLimit": 10,
+        "state": USState.California,
+    }
+    rv = put_json(client, f"/api/election/{election_id}/settings", settings)
+    assert_ok(rv)
+
+
+@pytest.fixture
+def contest_ids(
+    client: FlaskClient, election_id: str, jurisdiction_ids: List[str]
+) -> List[str]:
+    contests = [
+        {
+            "id": str(uuid.uuid4()),
+            "name": "Contest 1",
+            "isTargeted": True,
+            "choices": [
+                {"id": str(uuid.uuid4()), "name": "candidate 1", "numVotes": 600,},
+                {"id": str(uuid.uuid4()), "name": "candidate 2", "numVotes": 400,},
+            ],
+            "totalBallotsCast": 1000,
+            "numWinners": 1,
+            "votesAllowed": 1,
+            "jurisdictionIds": jurisdiction_ids,
+        },
+    ]
+    rv = put_json(client, f"/api/election/{election_id}/contest", contests)
+    assert_ok(rv)
+    return [str(c["id"]) for c in contests]
```

Comment:
does these two fixtures need to be overridden? seems like it would be fine to just use the default settings and contests

Reply:
I will check. If I recall I was getting some errors from Minerva when there were two contests.